tests/stack/test_api.py

```diff
@@ -140,6 +143,7 @@
                          ]
 }
 
+job_id_deleted = 111
```

Comment:
Maybe put as a class state (like `self.job_id_deleted`). Nit: change the naming to `deleted_job_id`?